### PR TITLE
Rename refactoring: test all possible occurrences

### PR DESCRIPTION
**Pull request by toinehartman**
Instead of manually writing near-duplicate tests that test several possible cursors, treat the expected rename locations as possible cursor inputs. Certain occurrences are currently not supported as cursors. For this we provide an optional set of `skipCursors`; these occurrences are not used as inputs, but are still expected to be in the output set.

Adds features to #397.

### CODE REVIEW

**Review by rodinaarssen — APPROVED** [review]
I like the fact that there there are no longer tests with nearly identical bodies, and that the test framework now test all occurrences by default.

Are the changes in `Rename.rsc` and `WorkspaceInfo.rsc` related to the rest of the PR, or are they unrelated bug fixes?

**Comment by toinehartman** [comment]
> Are the changes in Rename.rsc and WorkspaceInfo.rsc related to the rest of the PR, or are they unrelated bug fixes?

@rodinaarssen They are bug fixes that followed from the increased test coverage, so, thanks again for the suggestion for this improvement!

**Review by DavyLandman — APPROVED** [review]

**Comment by toinehartman** [comment]
Waiting for #427 to merge so we can verify this PR before merging.

**Pull request merged**
